<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmBaixaFrascos</class>
 <widget class="QDialog" name="frmBaixaFrascos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Baixa Frascos</string>
  </property>
  <widget class="QDateEdit" name="txtData">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>151</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbOperador">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>221</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>93</y>
     <width>99</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Fungo</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>93</y>
     <width>99</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Bactéria</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>93</y>
     <width>99</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Oxidação</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>93</y>
     <width>99</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Plantio</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="tblFrascos">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>621</width>
     <height>251</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="foreground">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>85</red>
       <green>87</green>
       <blue>83</blue>
      </color>
     </brush>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Merc</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Lote</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Clone</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nº Frasco</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Motivo</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nº Mds</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>54</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 205);</string>
   </property>
   <property name="text">
    <string>Data</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>60</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 205);</string>
   </property>
   <property name="text">
    <string>Operador</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>60</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 205);</string>
   </property>
   <property name="text">
    <string>Motivo</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>60</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Frascos</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btBaixar">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>490</y>
     <width>100</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>[ F12 ] Baixar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>80</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Nº de frascos:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblNFrascos">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>490</y>
     <width>54</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtCodigo">
   <property name="geometry">
    <rect>
     <x>102</x>
     <y>150</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans CJK HK</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);
background-color: rgb(249, 245, 184);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>621</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 205);</string>
   </property>
   <property name="text">
    <string>Código de Barras</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QProgressBar" name="pbBaixa">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>490</y>
     <width>371</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>0</y>
     <width>151</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>images/barcode_scanner.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblMsg">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>491</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_5">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>93</y>
     <width>99</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Repicagem</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtData</tabstop>
  <tabstop>cmbOperador</tabstop>
  <tabstop>radioButton</tabstop>
  <tabstop>radioButton_2</tabstop>
  <tabstop>radioButton_3</tabstop>
  <tabstop>radioButton_4</tabstop>
  <tabstop>txtCodigo</tabstop>
  <tabstop>tblFrascos</tabstop>
  <tabstop>btBaixar</tabstop>
 </tabstops>
 <resources>
  <include location="images/barcodescaner.qrc"/>
 </resources>
 <connections/>
</ui>
